lib/services/v2.0/nomination_pools/txs/set_commission_change_rate.dart

```diff
@@ -1,7 +1,6 @@
 import 'package:karma_coin/services/v2.0/nomination_pools/types.dart';
 import 'package:karma_coin/services/v2.0/txs/tx.dart';
 
-/// @HolyGrease who gets the commission? The pool creator?
```

Comment:
There is special account for this called `beneficiary` you can find it in `Commission` class of the `Pool` field `Commission commission;`